sda-commons-web-autoconfigure/src/main/java/org/sdase/commons/spring/boot/web/logging/HttpRequestLoggingInterceptor.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2022- SDA SE Open Industry Solutions (https://www.sda.se)
+ *
+ * Use of this source code is governed by an MIT-style
+ * license that can be found in the LICENSE file or at
+ * https://opensource.org/licenses/MIT.
+ */
+package org.sdase.commons.spring.boot.web.logging;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+import org.springframework.web.servlet.handler.HandlerInterceptorAdapter;
+
+@Component
+public class HttpRequestLoggingInterceptor extends HandlerInterceptorAdapter {
+
+  private static final Logger logger = LoggerFactory.getLogger(HttpRequestLoggingInterceptor.class);
+
+  @Override
+  public boolean preHandle(
+      HttpServletRequest request, HttpServletResponse response, Object handler) {
+    return true;
+  }
+
+  @Override
+  public void afterCompletion(
+      HttpServletRequest request, HttpServletResponse response, Object handler, Exception ex) {
+    logger.info(
+        String.format(
+            "%s %s%s %d - %s",
+            request.getMethod(),
+            request.getContextPath(),
+            request.getServletPath(),
+            response.getStatus(),
+            request.getHeader("user-agent")));
```

Comment:
Is there a reason not to use the String building capabilities of SLF4J?

```suggestion
    logger.info(
        "{} {}{} {} - {}",
        request.getMethod(),
        request.getContextPath(),
        request.getServletPath(),
        response.getStatus(),
        request.getHeader("user-agent"));
```